console/console-init/ui/src/modules/device/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
no need to export the component from 'device'

Reply:
Exporting it here just to keep it consistent with structure of other modules.